packages/api-plugin-promotions/src/handlers/applyPromotions.test.js

```diff
@@ -0,0 +1,69 @@
+import mockContext from "@reactioncommerce/api-utils/tests/mockContext.js";
+import applyImplicitPromotions from "./applyPromotions.js";
+
+const testTrigger = jest.fn().mockReturnValue(Promise.resolve(true));
```

Comment:
Whenever we mock an async function it's more concise to use `mockResolvedValues`. Also in multiple places in the unit test, we are returning sync value although the method is supposed to return a Promise. It's good to review the tests and replace `mockReturnValue` with `mockResolvedValue` especially if the returned value is not wrapped in a `Promise.resolve(value)` because we might be skipping an `await` somewhere an a bug might not be watched by tests.